velox/functions/sparksql/String.h

```diff
@@ -1130,4 +1130,52 @@ struct ReplaceFunction {
   }
 };
 
+template <typename T>
+struct FindInSetFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int32_t>& result,
+      const arg_type<Varchar>& str,
+      const arg_type<Varchar>& strArray) {
+    int32_t index = 1;
+    int32_t lastComma = -1;
+
+    if (std::string_view(str).find(',') != std::string::npos) {
+      result = 0;
+      return;
+    }
+
+    auto arrayData = strArray.data();
+    size_t arraySize = strArray.size();
+    size_t matchSize = str.size();
+
+    for (int i = 0; i < arraySize; i++) {
+      if (arrayData[i] == ',') {
+        if (i - (lastComma + 1) == matchSize &&
+            str ==
```

Comment:
why not use std::memcmp?

Reply:
Thank you for reivew, This is a good choice, The performance should be better using memcmp, and I will change to this way in a moment.I also did some tests and the results showed that std::memcmp performed better.Here are some of my test results.

`test group 1`
memcmp cost time : 	8.2309s
memcmp cost time : 	8.12958s
memcmp cost time : 	8.08002s
compare cost time : 	9.72118s
compare cost time : 	9.80633s
compare cost time : 	9.95188s

`test group 2`
memcmp cost time : 	15.6135s
memcmp cost time : 	15.7243s
memcmp cost time : 	15.6123s
compare cost time : 	18.8385s
compare cost time : 	19.0486s
compare cost time : 	19.1677s